v2/yarpcgrpc/headers.go

```diff
@@ -66,17 +66,13 @@ const (
 	EncodingHeader = "rpc-encoding"
 	// ErrorNameHeader is the header key for the error name.
 	ErrorNameHeader = "rpc-error-name"
+	// ErrorDetailsHeader is the header key for the error details used in gRPC.
+	// https://sourcegraph.com/github.com/grpc/grpc-go/-/blob/internal/transport/http_util.go#L317-326
```

Comment:
if this is the best we've got, let's use a permalink instead (includes the hash)